modules/ROOT/pages/_partials/mule-runtime-metrics.adoc

```diff
@@ -195,14 +215,14 @@ Example:
 
 // tag::muleMessages[]
 [[mule-messages]]
-== Mule Messages
+=== Mule Messages
 
 A _Mule message_ is the data (the payload and its attributes) that passes through one or more Mule flows in an application. A Mule message is part of a Mule event, which is generated when the event source within a Mule flow is triggered. For example, a Mule event that consists of a Mule message is created when an HTTP listener receives a request or each time the scheduler component triggers an execution of the Mule flow. 
 Mule message processors in a Mule flow (such as core components, file read operations, or the HTTP request operations) can then retrieve, set, and process Mule message data that resides in the Mule event according to their configurations. 
 A Mule message is immutable, so every change to a Mule message results in the creation of a new instance. Each processor in a flow that receives a Mule message returns a new Mule message that consists of a message payload (the body of the message) and message attributes (metadata associated with the message).
 
 [[billable-mule-messages]]
-=== Mule Message Scenarios that Count Towards Your Anypoint Platform Package Allotment
+==== Mule Message Scenarios that Count Towards Your Anypoint Platform Package Allotment
```

Comment:
```suggestion
==== Mule Message Scenarios That Count Towards Your Anypoint Platform Package Allotment
```

Reply:
I just looked up the difference between toward and towards. It's like grey/gray! Toward is US/Canada.

Reply:
Oh, interesting!

Reply:
Fixed in usage-metrics